po/fr.po

```diff
@@ -681,60 +679,55 @@ msgid "Preferences"
 msgstr "Préférences"
 
 msgid "Installation options:"
-msgstr ""
+msgstr "Options d'installation :"
 
 msgid "Try to remove libraries before (re)installing them?"
-msgstr ""
+msgstr "Essayer de supprimer les bibliothèques avant de les (ré)installer ?"
 
 msgid "Show README of newly installed libraries (if present)?"
-msgstr ""
+msgstr "Afficher le README (si présent) des nouvelles bibliothèques ?"
 
 msgid "Should newly installed libraries be added to Pd's search path?"
-msgstr ""
+msgstr "Ajouter les nouvelles bibliothèques aux chemins de recherche de Pd ?"
 
 msgid "Platform settings:"
-msgstr ""
+msgstr "Réglages de plateforme :"
 
 #, tcl-format
 msgid "Default platform: %s"
-msgstr ""
+msgstr "Plateforme par défaut : %s"
 
 msgid "User-defined platform:"
-msgstr ""
+msgstr "Plateforme définie par l'utilisateur :"
 
 msgid "Hide foreign architectures?"
-msgstr ""
+msgstr "Cacher les autres architectures ?"
 
-#, fuzzy, tcl-format
+#, tcl-format
 msgid "Deken %s Preferences"
-msgstr "Préférences"
+msgstr "Préférences de Deken %s"
 
-#, fuzzy
 msgid "[deken]: Start searching for externals..."
-msgstr "Installer des objets supplémentaires..."
+msgstr "[deken] : Recherche d'externals ..."
```

Comment:
Suggesting : "[deken] : Recherche d'externals ..." -> "[deken] : Démarrer la recherche d'externals ..."

Reply:
there, I chose "Démarrage de la recherche d'externals", because it's a debug message, not a choice to the user.

Reply:
ok